coalib/output/printers/LOG_LEVEL.py

```diff
@@ -1,7 +1,12 @@
+import logging
 from coalib.misc.Enum import enum
+from colorama import Fore
 
-LOG_LEVEL = enum("DEBUG", "INFO", "WARNING", "ERROR")
-LOG_LEVEL_COLORS = {LOG_LEVEL.ERROR: "red",
-                    LOG_LEVEL.WARNING: "yellow",
-                    LOG_LEVEL.INFO: "blue",
-                    LOG_LEVEL.DEBUG: "green"}
+LOG_LEVEL = enum(DEBUG=logging.DEBUG,
+                 INFO=logging.INFO,
+                 WARNING=logging.WARNING,
+                 ERROR=logging.ERROR)
```

Comment:
changing the log level could be easily separately done in an own commit right? (This one's too big IMO.)